stock_release_channel_process_end_time/models/stock_picking.py

```diff
@@ -69,6 +69,9 @@ def _after_release_set_expected_date(self):
                 rec.release_channel_id
                 and rec.release_channel_id.process_end_date
                 and rec.scheduled_date != rec.release_channel_id.process_end_date
+                # freeze scheduled_date in some cases don't need to change scheduled_date
+                # like in the test environemnt
+                and not self.env.context.get('freeze_scheduled_date')
```

Comment:
@anothingguy Nope. Introducing something only for tests is bad.

Instead, I would have added a configuration parameter boolean in release channel configuration to enable the date change at release.

So, in tests, this can be tested with and without the feature.

Reply:
I've just added a configuration parameter boolean in release channel configuration instead of using context

Reply:
Hi @rousseldenis, could you take a look at it since it's breaking the tests in `stock_available_to_promise_release` module

Reply:
Yes.

But where is it broken as 16.0 branch is green ?

Reply:
yes, it's broken the tests [here](https://github.com/OCA/wms/actions/runs/6194025712/job/16816412676?pr=750#step:8:338) in my module which needs to depends on `stock_release_channel_process_end_time`. I'm not sure why your tests was green previous time, but I did run the tests with your module in the local and failed as well. seems odd.